docs/content/manual/_incoming/vm-availability-when-harvester-upgrade-failed.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Check the VM is available when Harvester upgrade failed
+---
+
+## Category: 
+* Harvester Upgrade
+
+## Verification Steps
+1. Prepare the previous stable Harvester release cluster
+1. Create image
+1. Enable Network and create VM
+1. Create seveal virtual machine
```

Comment:
Should be "several"

Reply:
Appreciated for the check, just correct this typo.